src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -129,10 +123,10 @@ const MembersSubsection = ({
         )}
       </div>
     ),
-    [isContributorsSubsection, membersPageRoute, colonyWatchers, name],
+    [isContributorsSubsection, membersPageRoute, members, name],
   );
 
-  if (!colonyWatchers.length) {
+  if (!members) {
```

Comment:
For the reason I mentioned in my previous comment (the fallback for contributors/watchers), I don't think this will ever be true

Reply:
Yes, makes sense, updating to `!members.length` to check for an empty array instead